Buff Cybran navy's missile deflectors

lL1l1 opened the pull request:
## Issue
[Discord balance team thread](https://discord.com/channels/197033481883222026/1314781654841299014). Cybran navy is unable to properly defend itself against missile ships after they were changed to deflector type TMD with heavily nerfed fire rate and range.

## Description of the proposed changes
Increase the fire rate and range to match standard naval TMD.

Cybran cruiser/carrier TMD:
- Range: 20/26 -> 60
- Max target height: 8/10 -> infinite
- Reload time: 4/2.5 seconds -> 1 second

## Testing done on the proposed changes
Cybran cruisers/carriers can now defend fully against enemy missile cruisers. They can even kill UEF cruisers over some time, so it's not like the missiles being sent back are completely useless.

## Checklist
~~- [ ] Changes are annotated, including comments where useful~~
- [x] Changes are documented in the changelog for the next game version

lL1l1 committed:
Buff Cybran navy's missile deflectors

lL1l1 committed:
Create balance.6744.md

Garanas commented:
This is a **massive** buff for a unit that ignores the health of the missiles it intercepts. I'm fine either way, but maybe first increase the range to 40 and half the fire rate?

lL1l1 committed:
Restore deflector pitch range fully to zapper spec

lL1l1 committed:
Slow reload to 1.9 seconds

19 ticks lets it slowly push back Seraphim cruiser missiles which fire every 20 ticks, which allows Cybran to push into Seraphim cruisers
The target check interval has to be 0.1 for this 1 tick fire rate interval difference to work.

lL1l1 committed:
Update balance.6744.md

lL1l1 committed:
Reduce range a bit

Range has to be very large because this is navy, but it can be slightly smaller to slightly help high HP missiles (Torrent, Governor) compare fairly against Cybran navy compared to Sera/UEF navy.

lL1l1 commented:
- Reload 1.0 -> 1.9 seconds. This way it can push back against Sera's 1 HP, 2s reload missiles. It still defends against UEF's volley of 4 2 HP missiles spaced 1.5 seconds apart.
- Range 60 -> 44 which lets it defend a spread out navy but still gives some extra opportunity to hit stuff compared to gun TMD since it ignores missile HP.

I tested a small formation of frigates around the cruiser and groundfired too to see how far away from the cruiser the TMD of Cybran compared to Sera can defend.
I also tested Sera cruiser and saw that it lets UEF cruiser missiles get quite close, but I can't really compromise on range or fire rate to make the equivalent in Cybran.

Rhaelya reviewed (approved):
Everything looks in order
